.scrutinizer.yml

```diff
@@ -0,0 +1,16 @@
+checks:
+    php: true
+
+filter:
+    excluded_paths:
```

Comment:
Why would you want to exclude folders with actual PHP code ? I think we should strive to improve the entire code base, including tests and samples.

I'd remove the entire `filter` section